Added Star-wars font and Stary background

ayush268 opened the pull request:
The site design is made according to the theme - "Star-Wars".
Still working on the things but created the basic layout.

milindl commented on app/assets/stylesheets/styles.scss:
```diff
@@ -1,5 +1,98 @@
+    url("star-wars-font/STARWARS.svg#STARWARS") format("svg");
+    font-weight:normal;
+    font-style:normal;
+}
```
Basically most browsers support TTF, so we do not need all the other formats, they may just increase the size. It would be good to delete them from here and the files themselves also)

milindl commented on app/assets/stylesheets/styles.scss:
```diff
@@ -1,5 +1,98 @@
+/* For stary background */
+
 $heading-font: 'Source Sans Pro', sans-serif;
+$star-wars: 'STARWARS';
```
Add an alternative font - maybe sans-serif. It would be loaded in the case where loading of star wars fonts fails due to internet connectivity.

milindl commented on app/assets/stylesheets/styles.scss:
```diff
@@ -25,9 +118,9 @@ html {
         flex-flow: column nowrap;
         align-items: stretch;
-        background: #f7feff;
+        background: radial-gradient(ellipse at bottom, #1B2735 0%, #090A0F 100%);
```
we may need to add vendor specific prefixes to radial gradient(-moz and -webkit) and also specifiy a background color if it fails to render. IE support is as usual, lagging behind, but in this case I think the alternative is very simple (adding bgcolor) and we should do it.

milindl commented on app/assets/stylesheets/styles.scss:
```diff
@@ -25,9 +118,9 @@ html {
+        background: radial-gradient(ellipse at bottom, #1B2735 0%, #090A0F 100%);
         header {
-            background: #fff;
+            background: #1a1a1a;
```
Background needs to be darker here, it is too light

milindl commented on app/assets/stylesheets/styles.scss:
```diff
@@ -37,10 +130,10 @@ html {
+                    color: white;
                 }
             }
             .hamburger {
```
remove the hamburger class entirely when you're at it...just for cleanup purposes, not related to your change.

milindl commented on app/views/layouts/_header.html.erb:
```diff
@@ -1,6 +1,6 @@
 <header>
     <div class ="logocontainer">
-        <%= link_to "Leaderboard", root_path,id: "logo" %>
+        <%= link_to "STAR BOARD", root_path,id: "logo" %>
```
Can we make it one word please? Like starboard.
Also, we need to have "PClub Leaderboard" somewhere, maybe as a small subheading, so that we can change the theme at the end after a few years.

milindl commented on app/assets/stylesheets/styles.scss:
```diff
@@ -172,12 +266,11 @@ html {
             margin: 0 auto;
             padding: 0px;
-            background: $blue;
+            background: url('https://static-mh.content.disney.io/starwars/assets/footer/vader-1c0fe900f0e4.jpg') no-repeat center, #1a1a1a;
```
Background again needs to be darker. Please choose a shade, I can make the background with that shade.

ayush268 commented:
Fixed:
Background repeats (But limit is set to 200vmax).
Other font files except TTF removed.
Added an alternative font.
Background color rendering issues in cases of different browsers and their features.
Removed the hamburger class.
Need to show PClub leaderboard somewhere.
Colors are to be changed.
Text need to be on opaque bg-background like star wars-site.
Background needs to be darker.
Footer background also needs to be darker.

Need to Fix:
Breaks in bottom-border in Standings table.

Added:
webkit and moz for transitions also (Thought of Internet Explorer).
Small tweaks.

milindl commented:
Hey, this is merged now, great work on this! 

👍
